### DISCUSSION TITLE
Welcome to Python_BMP Discussions!

### DISCUSSION THREAD

**Discussion by TechnoTanuki** [TechnoTanuki/Python_BMP#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TechnoTanuki** [comment]
hi

**Comment by MrToughFB** [comment]
Hello, how do you create a proxy site in python?

**Reply by TechnoTanuki** [reply]
this was meant for discussions related to this project.. try joining Python discord and asking there
